sync-service/src/main/resources/application-local.yaml

```diff
@@ -0,0 +1,4 @@
+stayhome:
```

Comment:
I think this should use environment variables and not be defined in a static file.

Reply:
in generell using spring-boot: every property can always be overrriden by an environment-variable.